Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM python:3.7-buster as base
+
+ENV PATH="/root/.poetry/bin:$PATH"
+
+
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+
+COPY poetry.lock pyproject.toml ./
+
+RUN poetry config virtualenvs.create false
+
+FROM base as production
+
+RUN poetry install --no-dev
+
+WORKDIR /app
+COPY . /app
```

Comment:
This is fine, but if we take a fairly broad approach of copying then we should make sure we're not including anything accidentally - and particularly secrets! As such, we should add a `.dockerignore` file with some sensible contents, e.g. it would probably include at least
```
.env
.venv
# maybe some caching folders, like **/__pycache__
# ...
```